CS410_Assignment2-master/src/SensorApplication.java

```diff
@@ -0,0 +1,41 @@
+
+import java.awt.Dimension;
+import java.awt.GridLayout;
+import javax.swing.JFrame;
+
+import adaptors.PressureAppAdaptor;
+import adaptors.RadiationAppAdaptor;
+import adaptors.TemperatureAppAdaptor;
+import sensor.PressureSensor;
+import sensor.RadiationSensor;
+import sensor.TemperatureSensor;
+
+	public class SensorApplication extends JFrame {
+		
+		public SensorApplication() {
+			setTitle("Sensor Tracker");
+			setLayout(new GridLayout(3,1));
+		
+			//Construct all sensor panels, with each given an adapted version of the sensor
```

Comment:
I do not believe critical and danger ranges should be hardcoded into the application, but instead be part of the Adapters as another function.